app/components/bali/calendar/component.html.erb

```diff
@@ -0,0 +1,46 @@
+<div class="calendar-component">
+  <div class="box <%= period == :month ? 'month-view' : 'week-view' %>">
+    <%= header %>
+
+    <div class="table-container">
+      <table class="table is-fullwidth">
+        <thead>
+          <tr>
+            <% date_range.slice(0, all_week ? 7 : 5).each do |day| %>
+              <th class="has-text-centered has-text-grey-dark has-text-weight-normal is-size-4">
+                <%= t('date.day_names')[day.wday] %>
```

Comment:
I'm pretty sure this is going to raise, not sure, but there is no translation on Bali by itself, if you integrate with your host app will work fine, but in Bali Standalone im pretty sure not..

https://github.com/Grupo-AFAL/ga-apps/blob/573a43a7e395b7f269c5efe3d1d20cd69718aacc/config/locales/defaults/es.yml#L38

Reply:
I will raise an error only if the app has the configuration to raise errors on missing translations, although the default is not to raise.

This is the default rails translation for dates. So I think it's good to asume that the host app will have those translations.